pkg/controller/stacks/stack/stack.go

```diff
@@ -818,6 +822,18 @@ func (h *stackHandler) prepareDeployment(d *apps.Deployment) {
 	meta.AddLabels(d, labels)
 
 	d.Spec.Template.Spec.ServiceAccountName = h.ext.Name
+
+	if h.forceImagePullPolicy != "" {
+		for _, c := range [][]corev1.Container{
+			d.Spec.Template.Spec.Containers,
```

Comment:
Does this change any behaviors for template stacks using the templating-controller?

Reply:
The templating controller is treated like any controller. Since the change that introduced `imagePullPolicy` as a StackInstall parameter, the templating controller adopts the supplied pull policy.  If the StackManager wants to enforce a value, that value should be enforced on the templating controller as well.